spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/spring/tx/TransactionObject.java

```diff
@@ -22,7 +22,7 @@
  * @author ameingast@gmail.com
  * @author Florian Kleedorfer
  */
-public class TransactionObject implements SmartTransactionObject {
```

Comment:
@jeenbroekstra The spring somponents module is not part of the core module. We haven't added the japicmp build plugin to the pom files either. Does that entail that we don't follow semantic versioning for the spring components?

Reply:
We do - we follow semantic versioning for the entire framework. 

I think we can let this slide "just this once" since the component is so new and is basically still in experimental stage. Might be good to explicitly mark it as such though.